### DISCUSSION TITLE
Error measuring code not showing plot

### DISCUSSION THREAD

**Discussion by Xapp00** [Part-II-Computational-Physics/.github#13] [Q&A]
Hi I've been having trouble running the code that measures the error made by Lax-Wendroff, no errors were reported but the code could only produce a blank graph and then kept loading. (I have done the sanity check and I think my code for Lax-Wendroff works ok, with stability as expected etc.)

**Comment by NathanMagnan** [comment]
The graph can take some time to appear because it needs to solve a few high-resolution cases. This could explain the long loading. The blank plot could be the axes being plotted before the lines.

Have you tried to print something at the end of each loop of the error measuring code (or other debugging approaches)?
Does the code work if you try to measure the error of the explicit upwind scheme?

**Reply by Xapp00** [reply]
Yes sorry for the question, it does load in the end it just took ages. Thanks!
